util/dvsim/LsfLauncher.py

```diff
@@ -202,20 +194,18 @@ def launch(self):
             "-J",
             job_array,
             "-oo",
-            "{}.%I.out".format(job_script_wo_idx),
+            "{}.%I.out".format(job_script),
             "-eo",
-            "{}.%I.out".format(job_script_wo_idx)
+            "{}.%I.out".format(job_script)
         ]
 
         if job_rusage:
             cmd += ["-R", job_rusage]
 
-        cmd.append(
-            shlex.quote(
-                "/usr/bin/bash {} $LSB_JOBINDEX".format(job_script_wo_idx)))
+        cmd += ["/usr/bin/bash {} $LSB_JOBINDEX".format(job_script)]
 
         try:
-            p = subprocess.run(' '.join(cmd),
+            p = subprocess.run(shlex.join(cmd),
```

Comment:
If you're touching this again, I'd really suggest trying what @imphil said in the previous PR. I *think* you misunderstood him and changed what's now on line 205 (which I agree wouldn't work). I believe that `subprocess.run(shlex.join(args), shell=True)` is always equivalent to `subprocess.run(args, shell=False)` (assuming that you're not trying to use a shell alias or something). Maybe try that?

Reply:
How odd! Well, the version you've got now looks right to me. Thanks for getting to the bottom of things.